.github/workflows/ci.yml

```diff
@@ -397,7 +400,7 @@ jobs:
         timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
         run: |
           bash ./scripts/ci/init.sh
-          bash ./scripts/ci/build.sh
+          BUILD_OPTIONS="-Ddocker.buildArg.JRE_VERSION=${{ matrix.jre-version }} ${BUILD_OPTIONS}" bash ./scripts/ci/build.sh
```

Comment:
Could split the `BUILD_OPTIONS` assignment and the script invocation on separate lines for readability.